posts/inside-rust/2020-06-25-governance-wg

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "Governance Working Group Update: Meeting 25 June 2020"
+author: Val Grimm
+team: The Governance WG <https://github.com/rust-lang/wg-governance>
+---
+
+Hello everyone! 
+
+We held another meeting of the Rust Governance Working Group on Zulip on 25 June.
+Agenda: Clarify taxonomy of groups in  the Rust Project
+
+You can find the [detailed minutes](https://github.com/rust-lang/wg-governance/blob/master/minutes/2020.06.25.md) 
+on the [wg-governance](https://github.com/rust-lang/wg-governance) repository, but here is a quick summary: 
+- Going forward there will be two types of groups in the Rust organization
+    * Teams
+    * Project groups parented by those teams, have a charter, do not have final decision-making power
+    * Subteams that refine the work of their parent team (e.g. community/events or compiler/prioritization)
+    * Subteams cover a defined area, they are not trying to create a project
+- The Rust Project will not list or link to any community groups, or keep community projects on its Github
+    * If community groups want folks to find them they can create a group on the Mozilla Community Portal, which the Rust Project will link to instead of any individual groups
```

Comment:
Same as above I feel this is a little too definitive and i think should be a bit more framed as a discussion.
```suggestion
- As part of this we also discussed the project's relationship with developer groups in the community, and should and if so how would the Rust project promote those groups.
- There was a feeling that the Rust project should not act as a primary or secondary source for community groups, and should instead promote secondary resources like the [Mozilla Community Portal][mozcp] as a way to help people find groups in the community.

[mozcp]: https://community.mozilla.org
```